<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWin</class>
 <widget class="QWidget" name="SetupWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetupWin</string>
  </property>
  <widget class="QPushButton" name="pbGain">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Gain</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDateTime">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>DateTime</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbRefRange">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref Range</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPrint">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbParaUnit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Para Unit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAutoMaintain">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Maintain</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDepartment">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Department</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOperator">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Operator</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDoctor">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Doctor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMachine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Machine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCalFactor">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Calibrate Factor</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
